include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -130,44 +101,25 @@ using __maximum = sycl::maximum<_T>;
 template <typename _T = void>
 using __minimum = sycl::minimum<_T>;
 
-#else  // _ONEDPL_SYCL2020_FUNCTIONAL_OBJECTS_PRESENT
-template <typename _T>
-using __plus = sycl::ONEAPI::plus<_T>;
-
-template <typename _T>
-using __maximum = sycl::ONEAPI::maximum<_T>;
-
-template <typename _T>
-using __minimum = sycl::ONEAPI::minimum<_T>;
-#endif // _ONEDPL_SYCL2020_FUNCTIONAL_OBJECTS_PRESENT
-
 template <typename _Buffer>
 constexpr auto
 __get_buffer_size(const _Buffer& __buffer)
 {
-#if _ONEDPL_LIBSYCL_VERSION >= 50300
     return __buffer.size();
-#else
-    return __buffer.get_count();
-#endif
 }
 
 template <typename _Accessor>
 constexpr auto
 __get_accessor_size(const _Accessor& __accessor)
 {
-#if _ONEDPL_LIBSYCL_VERSION >= 50300
     return __accessor.size();
-#else
-    return __accessor.get_count();
-#endif
 }
 
 template <typename _Item>
 constexpr void
 __group_barrier(_Item __item)
 {
-#if 0 //_ONEDPL_LIBSYCL_VERSION >= 50300
+#if 0
```

Comment:
@MikeDvorskiy  what we should to do with this `#if 0` ?

Reply:
//TODO: usage of sycl::group_barrier: probably, we have to revise SYCL parallel patterns which use a group_barrier.
    // 1) sycl::group_barrier() implementation is not ready
    // 2) sycl::group_barrier and sycl::item::group_barrier are not quite equivalent

Reply:
`sycl::group_barrier()` did not work at that moment

Reply:
@MikeDvorskiy, could you please explain, what exactly did not work and in which cases?

Reply:
Issue created: https://github.com/oneapi-src/oneDPL/issues/1679
let's fix it in the separate PR later.